splinepy/microstructure/tiles/double_lattice.py

```diff
@@ -57,19 +57,19 @@ def create_tile(
         # set to default if nothing is given
         if parameters is None:
             self._logd("Tile request is not parametrized, setting default 0.2")
-            parameters = _np.ones((1, 2)) * 0.1
-        # Maintain backwards compatibility
-        elif parameters.shape[1] == 1:
-            _warning("DoubleLattice now expects 2 values")
-            index_second_value = 0
-            self._n_info_per_eval_point = 1
+            parameters = _np.ones((1, self._n_info_per_eval_point)) * 0.1
         if not (
             _np.all(parameters > 0)
             and _np.all(parameters < 0.5 / (1 + _np.sqrt(2)))
         ):
             raise ValueError(
                 "Parameters must be between 0.01 and 0.5/(1+sqrt(2))=0.207"
             )
+        # Maintain backwards compatibility
+        if parameters.shape[1] == 1:
+            _warning("DoubleLattice now expects 2 values")
+            index_second_value = 0
+            self._n_info_per_eval_point = 1
```

Comment:
Please delete this section! It's not that it would not work, it's just avoidable

Reply:
I don't know how to avoid this 🥲